proof/refine/ARM_HYP/Retype_R.thy

```diff
@@ -4759,6 +4766,18 @@ lemma createNewCaps_vms:
                      ARM_HYP_H.toAPIType_def object_type.splits)
   done
 
+(* FIXME arch-split: move, use as alternative when mask_range comes up *)
+lemma new_range_subset':
+  assumes
+        cover: "range_cover ptr sz (objBitsKO ko) n"
+    and addr: "x \<in> set (new_cap_addrs n ptr ko)"
+  shows       "mask_range x (objBitsKO ko) \<subseteq> {ptr .. (ptr && ~~ mask sz) + 2 ^ sz - 1}"
+  using assms
+  apply -
+  apply (drule (1) new_range_subset)
+  apply (simp add: mask_def add_diff_eq)
+  done
```

Comment:
Is this moved from somewhere? The indentation is a bit strange and the `assumes`/`shows` is unnecessary, because not used in the proof. Removing that would also get rid of the `apply -`.

Reply:
This is a quick copy-pasta of `new_range_subset` with the definition changed to use mask, to smooth over old phrasings and new. I'll tweak it.